inference-engine/src/transformations/include/transformations/common_optimizations/division_to_zero_fp16_resolver.hpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <utility>
+#include <memory>
+
+#include <transformations_visibility.hpp>
+#include <ngraph/pass/graph_rewrite.hpp>
+#include "ngraph/pattern/matcher.hpp"
+
+namespace ov {
+namespace pass {
+
+class TRANSFORMATIONS_API DivisionToZeroFP16Resolver;
```

Comment:
nit: `DivisionByZero` ?

Reply:
You are right. Grammatically division by is correct. Updated